setup.py

```diff
@@ -14,7 +14,7 @@
     changelog = changelog_file.read()
 
 install_requirements = [
-    'squid-py==0.2.14',
+    'squid-py==0.2.17',
```

Comment:
When this changes to 0.2.19 it is likely many other things will have to change (e.g. configs, etc.)